templates/errors/session-expired.html

```diff
@@ -19,7 +19,9 @@ <h1 {{helpers.track('visible', 'Timeout', 'Timeout expired')}}>{{ _("Your sessio
   {% endcall %}
 
   {% if account_service_url %}
-    <a href="{{account_service_url}}">{{ _("Return to your account") }}</a>
+    <p>{{ _("We have saved your progress and you will need to ") }}
```

Comment:
This should help make the translation of this more meaningful. At the moment, they are split to 3 difference sentence which won't make sense?
```html
  <p>
    {{ _('We have saved your progress and you will need to %(enter_your_unique_code)s to access the survey again.', enter_your_unique_code=('<a href="' ~ account_service_url ~ '">%s</a>' % _('enter your unique code'))|safe) }}
    </p>
```

Reply:
Updated this.

Reply:
As discussed:
```
    {{ _('We have saved your progress and you will need to <a href="%(account_service_link)s">enter your unique code</a> to access the survey again.', account_service_link=account_service_url) }}
```